hieradata/role/jenkins.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 psick::base::linux_classes:
   jenkins: psick::jenkins
+  ssh: role::ssh
```

Comment:
Why is this `psick::base::linux_class` with a `role` prefix not a `psick` prefix? @alvagante Comments?

Reply:
That’s correct in general terms. 
We specify the class name we want to include , so it can be psick::ssh or openssh (using an existing openssh module) or profile::ssh , or even role::ssh even if in puppet world this is not a proper term, since i assume this looks more like a profile to manage ssh

Reply:
We want to use the proper terms. This profile is designed to enable multiple ssh keys on the server. Would you suggest a name like `profile:ssh-multiple`? Is this implementation something that could or should go into upstream?

Reply:
@alvagante I'll try to be clearer. This profile is intended to enable multiple ssh keys on the Jenkins slave, which are deploy keys for two different GitHub repositories. It's not intended for managing ssh keys for real users (although perhaps the distinction is irrelevant). What do you suggest?